src/ui/zcl_abapgit_gui_page_view_repo.clas.abap

```diff
@@ -585,29 +585,37 @@ CLASS ZCL_ABAPGIT_GUI_PAGE_VIEW_REPO IMPLEMENTATION.
   METHOD constructor.
 
     DATA: lo_settings TYPE REF TO zcl_abapgit_settings,
+          lx_error    TYPE REF TO zcx_abapgit_exception,
           lv_package  TYPE devclass.
 
-    super->constructor( ).
```

Comment:
to be on the safe side, suggest keeping `super->constructor( ).`

Reply:
good catch. it was deleted by accident.